components/map/map-status.tsx

```
@@ -0,0 +1,21 @@
+'use client'
+
+import { Spinner } from '@/components/ui/spinner'
+import { Map } from 'lucide-react'
```

Biome (2.3.13)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)